Source/CarthageKit/Dependency.swift

```diff
@@ -27,6 +32,23 @@ public enum Dependency {
 		}
 	}
 
+	/// The unique identifier for this project which is useful as a file system path.
+	public var fileSystemIdentifier: SignalProducer<String, CarthageError> {
+		let task = Task("/usr/bin/shasum", arguments: ["-a", "256"])
```

Comment:
I hate shelling out for things like this. But I guess there's probably not an easy Swift option. 😞